### PR TITLE
Normalize range/version used in maxSatisfying method

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Connects to #53

change-type: minor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,140 +106,6 @@ If both values are invalid semver values, then the values are compared alphabeti
 | versionA | <code>string</code> \| <code>null</code> \| <code>undefined</code> | The first version to compare |
 | versionB | <code>string</code> \| <code>null</code> \| <code>undefined</code> | The second version to compare |
 
-**Example**
```
Comment (pimterry) [inline README.md]: This appears to have removed all the examples from the README?

File: test/index.spec.ts [hunk 2]
```diff
@@ -81,46 +91,91 @@ describe('resin-semver', () => {
...
+
+			expect(
+				semver.compare('Resin OS 2.0.0.prod', 'Resin OS 2.0.0+prod'),
+			).to.equal(0);
+			expect(
+				semver.compare('Resin OS 2.0.0.prod', 'Resin OS 2.0.0 (prod)'),
+			).to.equal(0);
+			expect(
+				semver.compare('Resin OS 2.0.0+prod', 'Resin OS 2.0.0 (prod)'),
+			).to.equal(0);
```
Comment (pimterry) [inline test/index.spec.ts]: I can see why this (and all the other changes in this file) have happened, and prettier is great, but this does make me a little sad. Could we boost the line length, so all these tests are readable (both here and in the docs), or is that not worth it?

Comment (LucianBuzzo) [reply test/index.spec.ts]: I'm happy to boost the line length here, readability is important for these test files

File: README.md (hunk 1 above)
Comment (LucianBuzzo) [reply README.md]: WAAAAH

**Pull request merged**